app/common/ui-helper/filters/momentFilter.js

```diff
@@ -13,14 +13,18 @@
         .filter("moment", () => filter);
 
     function filter(input, format) {
-        if (moment.isMoment(input)) {
+        var momentInput = input;
+        if (typeof input === 'string') {
```

Comment:
I think its possible to always wrap the input inside `moment` regardless of the type, eliminating the need for `if(moment.isMoment(...))`.

Reply:
We need the if so we don't try to process anything other than `string` and `moment` objects.